### ISSUE TITLE
Romanian translation

### ISSUE THREAD

**Issue by Alexandru51** [Lucchetto/SuperImage#34]
Hello, i want to translate this app in romanian.  (If possible)

**Comment by Alexandru51** [comment]
Alright then. But it is ok if you can provide the strings.xml file or something?

**Comment by Lucchetto** [comment]
Yes, I'm re-organizing the project. Once desktop version will be open source, you can submit strings there
